src/Microsoft.TemplateEngine.Cli/TemplateListResolutionResult.cs

```diff
@@ -215,9 +218,13 @@ public IReadOnlyList<ITemplateMatchInfo> GetBestTemplateMatchList(bool ignoreDef
         private IReadOnlyList<ITemplateMatchInfo> BaseGetBestTemplateMatchList(bool ignoreDefaultLanguageFiltering)
         {
             IReadOnlyList<ITemplateMatchInfo> templateList;
-
             if (TryGetUnambiguousTemplateGroupToUse(out templateList, ignoreDefaultLanguageFiltering))
             {
+                // Unambiguous template group could be a partial match
+                if (templateList.Count(x => (x.IsMatch || x.IsMatchExceptContext())) == 0)
```

Comment:
super minor: the parenthesis within the Count logical condition are not required & could be as follows...
``` templateList.Count(x => x.IsMatch || x.IsMatchExceptContext())```